models/sessions/scratch/snowplow_mobile_sessions_aggs.sql

```diff
@@ -45,33 +47,32 @@ with events as (
 
   from events e
 
-  group by 1
+  group by 1,2
 )
 
 , app_errors as (
   {% if var("snowplow__enable_app_errors_module", false) %}
     select
+      ae.session_identifier,
       ae.session_id,
       COUNT(distinct ae.event_id) AS app_errors,
       COUNT(distinct case when ae.is_fatal then ae.event_id end) as fatal_app_errors
 
     from {{ ref('snowplow_mobile_app_errors_this_run') }} ae
 
-    group by 1
+    group by 1,2
   {% else %}
     select
-      {% if target.type == 'postgres' %}
-        cast(null as uuid) as session_id,
-      {% else %}
-        cast(null as {{type_string() }}) as session_id,
-      {% endif %}
+      cast(null as {{type_string() }}) as session_identifier,
```

Comment:
Regardless of removing/keeping identifier in this table, keep this change because it might not be a uuid if they change it